src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/WiremockService.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
+import com.github.tomakehurst.wiremock.client.WireMock
+import org.springframework.stereotype.Service
+
+@Service
+class WiremockService(private val wireMockServer: WireMockServer) {
+
+  fun resetAllStubsAndMappings() {
+    wireMockServer.resetAll()
+  }
+
+  fun stubHmppsAuthUserMe(
+    username: String = "auser_gen",
+    displayName: String = "Albert User",
+    activeCaseLoadId: String = "BXI",
+  ) {
+    wireMockServer.stubFor(
+      WireMock.get(WireMock.urlPathMatching("/auth/api/user/me"))
+        .willReturn(
+          ResponseDefinitionBuilder.responseDefinition()
+            .withStatus(200)
+            .withHeader("Content-Type", "application/json")
+            .withBody(
+              """
+                {
```

Comment:
This is the very simple object that HMPPS Auth `/api/user/me` returns - for the purpose of what we are doing in this PR we only care about the `activeCaseLoadId` field